FakeItEasy.sln

```diff
@@ -74,6 +74,14 @@ Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "samples", "samples", "{2EED
 		samples\FakeItEasy.Examples.ruleset = samples\FakeItEasy.Examples.ruleset
 	EndProjectSection
 EndProject
+Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "analyzer", "analyzer", "{86990EDA-25B4-4BFC-BC44-14D1B754261A}"
+EndProject
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "FakeItEasy.Analyzer", "src\analyzer\FakeItEasy.Analyzer\FakeItEasy.Analyzer.csproj", "{F97E3B4B-080D-414A-9F00-EE20F4451E36}"
+EndProject
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "FakeItEasy.Analyzer.Test", "src\analyzer\FakeItEasy.Analyzer.Test\FakeItEasy.Analyzer.Test.csproj", "{38F3191B-1CE0-4EE0-930F-794A2F2F4CDB}"
+EndProject

```

Comment:
The solution and disk locations seem to be at odds (assuming the VSIX is going to be removed):

```
(Solution)
  analyzer
    FakeItEasy.Analyzer
  tests
    Unit
      FakeItEasy.Analyzer.Test

(Disk)
  src
    analyzer
      FakeItEasy.Analyzer    
      FakeItEasy.Analyzer.Test
```

I think it should be probably look like this:

```
(Solution)
  src
    FakeItEasy.Analyzer
  tests
    Unit
      FakeItEasy.Analyzer.Tests

(Disk)
  src
      FakeItEasy.Analyzer
  tests
      FakeItEasy.Analyzer.Tests
```

Note that `FakeItEasy.Analyzer.Tests` should also be pluralised.